packages/template-retail-react-app/app/components/store-locator/stores-list.jsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {useIntl} from 'react-intl'
+import PropTypes from 'prop-types'
+import {
+    AccordionItem,
+    AccordionButton,
+    AccordionIcon,
+    AccordionPanel,
+    Box
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+
+const StoresList = (props) => {
+    const {storesData} = props
+    const intl = useIntl()
+
+    var storesInfo = (storesData?.data?.data || []).sort((a, b) => a.distance - b.distance)
+    return storesInfo?.map((store, index) => {
```

Comment:
Not a big deal, but if you ensure `storesInfo` is always an array, then you won't need guards here.

You might also consider using `const` rather than `var`.

Reply:
Thanks I changed it to a `const`. It seems validation for `storesInfo` being `undefined` is necessary while the hook asynchronously loading information.